zerver/lib/push_notifications.py

```diff
@@ -770,6 +770,7 @@ def get_base_payload(user_profile: UserProfile) -> Dict[str, Any]:
     # These will let the app support logging into multiple realms and servers.
     data["server"] = settings.EXTERNAL_HOST
     data["realm_id"] = user_profile.realm.id
+    data["realm_url"] = user_profile.realm.url
```

Comment:
Cool, this is helpful: it's adding `realm_url` to the notification payloads, as an alias of `realm_uri`.

Unlike most of this commit, it's changing the server's behavior as seen by clients. So let's put it in its own commit, with its own commit message. That will help us track what server version this change happened in.

I'd also like to have this mentioned in the API changelog, which also means bumping the feature level for it. The notification payloads as a whole aren't yet documented in the API docs — that's one of the handful of notable areas that's still missing from the API docs — but even without overall documentation for them, it's useful to document the changes.

Reply:
I have a commit that addresses all the api endpoint changes. To address what you suggest, I have merge [the change of this api](https://github.com/zulip/zulip/blob/407b4b4172580c8ded9d181514d888089f43eb9d/zerver/lib/push_notifications.py#L779) into this commit as well. I also document this mobile notification API change in `/api_docs/changelog.md` ([here](https://github.com/zulip/zulip/blob/407b4b4172580c8ded9d181514d888089f43eb9d/api_docs/changelog.md?plain=1#L30-L31)).

Reply:
Please let me know if the change is appropriate - thanks!